### DISCUSSION TITLE
Inquiry on Projection Numbers for CT Reconstruction in Cone Beam and Fan-beam CT

### DISCUSSION THREAD

**Discussion by yangshiyan** [astra-toolbox/astra-toolbox#381] [General]
Dear Experts:

I am a freshman studying CT reconstruction. I recently came across a question regarding the number of projections required for CT reconstruction, particularly in the context of cone beam CT and Fan-beam CT. 
(1)	I observed that the projection number is typically set to 180 in the examples provided by the astar-toolbox. Dose this imply that constructing a CT with 180 projections is sufficient.
(2)	For the same phantom, is the result of a certain slice construction (such as CT number) obtained through cone beam 3D reconstruction the same as that obtained through Fan-beam 2D reconstruction.

Thank you very much for your time and consideration. I eagerly await your response. 

Warm regards.

**Comment by askorikov** [comment]
Hi! Question (1): the number of projections primarily determines the resolution of your reconstruction, which is quantitatively described by [Crowther criterion](https://en.wikipedia.org/wiki/Crowther_criterion). So in principle the more you have, the better (until other factors start to limit the resolution. Number 180 is just arbitrary. Question (2) I didn't quite get. Cone beam and fan beam are different geometries, and which one you would use is determined by the experimental setup used to obtain the data.
